README.md

```diff
@@ -38,10 +38,10 @@ make deploy
   ```
 </details>
 
-
 ### Installing via OLM
 
-To install the Operator using the [Operator Lifecycle Manager](https://olm.operatorframework.io/), you need to make the Operator CSVs available in the cluster by creating a `CatalogSource` resource.
+To install the Operator using the [Operator Lifecycle Manager](https://olm.operatorframework.io/), you need to make the
+Operator CSVs available in the cluster by creating a `CatalogSource` resource.
```

Comment:
I guess your IDE doesn't automatically word wrap markdown content while keeping it in a single line of code.

Reply:
Yes. It looks to add a hard break at 120 characters also when auto reformatting on Goland IDE 🤔 

Happy to revert these anyway if that's the preference. It's pretty minor